be/src/common/config.h

```diff
@@ -876,6 +876,10 @@ CONF_mInt32(spill_init_partition, "16");
 // The maximum size of a single log block container file, this is not a hard limit.
 // If the file size exceeds this limit, a new file will be created to store the block.
 CONF_Int64(spill_max_log_block_container_bytes, "10737418240"); // 10GB
+// The maximum size of a single spill directory, for some case the spill directory may
+// be the same with storage path. Spill will return with error when used size has exceeded
+// the limit.
+CONF_Int64(spill_max_dir_bytes, "214748364800"); // 200GB
```

Comment:
use can config multiple dirs for spill_local_storage_dir, should we also consider it here?